packages/sit-onyx/src/components/OnyxCalender/OnyxCalender.vue

```diff
@@ -0,0 +1,377 @@
+    t.value("days.wednesday.short"),
+    t.value("days.thursday.short"),
+    t.value("days.friday.short"),
+    t.value("days.saturday.short"),
+    t.value("days.sunday.short"),
+  ];
+  const longNames = [
+    t.value("days.monday.long"),
+    t.value("days.tuesday.long"),
+    t.value("days.wednesday.long"),
+    t.value("days.thursday.long"),
+    t.value("days.friday.long"),
+    t.value("days.saturday.long"),
+    t.value("days.sunday.long"),
+  ];
+
+  return calenderSize.value === "big" ? longNames : shortNames;
+});
+const calenderRef = useTemplateRef("calender");
+const {
+  currentYear,
+  currentMonth,
+  selectedDate,
+  weeks,
+  weekdays,
+  isToday,
+  isSelected,
+  isFocused,
+  isWeekend,
+  goToPreviousMonth,
+  goToNextMonth,
+  goToToday,
+  goToDate,
+  handleKeyNavigation,
+} = useCalendar({ ...props, dayNames, calenderRef });
+
+const { densityClass } = useDensity(props);
+
+const skeleton = useSkeletonContext(props);
+
+watch(selectedDate, (newDate) => {
+  if (newDate) {
+    emit("update:selectedDate", newDate);
+  }
+});
+
+const { width } = useResizeObserver(calenderRef);
+
+const sizeClass = computed(() => {
+  return `onyx-calender--${calenderSize.value}`;
+});
+</script>
+
+<template>
+  <div v-if="skeleton" :class="['onyx-component', 'onyx-calender-skeleton', densityClass]">
+    <OnyxSkeleton class="onyx-calender-skeleton__header" />
+    <OnyxSkeleton class="onyx-calender-skeleton__body" />
+  </div>
+
+  <div
+    v-else
+    ref="calender"
+    :class="[
+      'onyx-component',
+      'onyx-calender',
+      sizeClass,
+      densityClass,
+      { 'onyx-calender--disabled': disabled },
+    ]"
+  >
+    <div class="onyx-calender__header">
+      <div class="control-container time-control-container">
+        <OnyxSystemButton
+          v-if="calenderSize !== 'small'"
+          label="Today"
+          class="control-container__today-btn"
+          :disabled="disabled"
+          @click="goToToday"
+        />
+        <OnyxSystemButton
+          label="previous Month"
+          :icon="iconChevronLeftSmall"
+          :disabled="disabled"
+          @click="goToPreviousMonth"
+        />
+        <OnyxHeadline is="h2" class="control-container__date-display">
+          {{
+            new Date(currentYear, currentMonth).toLocaleDateString("de-DE", {
+              month: "long",
+              year: "numeric",
+            })
+          }}
+        </OnyxHeadline>
+        <OnyxSystemButton
+          label="next Month"
+          :icon="iconChevronRightSmall"
+          :disabled="disabled"
+          @click="goToNextMonth"
+        />
+      </div>
+      <div class="control-container">
+        <slot name="actions"></slot>
+      </div>
+    </div>
+    <div class="onyx-calender__body">
+      <!-- eslint-disable-next-line vuejs-accessibility/interactive-supports-focus -->
+      <table role="grid" aria-labelledby="calendar-label" @keydown="handleKeyNavigation">
```

Comment:
These and the other aria specific attributes and event-listener should be set through the composable. E.g.:
```suggestion
      <table v-bind="calendarGrid">
```